src/js/GlobalConstants.js

```diff
@@ -18,7 +18,8 @@ const globalConstants = {
     // Phase 1 release
     CARES_ACT_RELEASED: true,
     // Phase 2 release
-    CARES_ACT_RELEASED_2: true
+    CARES_ACT_RELEASED_2: true,
+    FILES_SERVER_BASE_URL: process.env.FILES_SERVER_BASE_URL
```

Comment:
recommend making this an inferred property on this object based on the ENV rather than requiring this to be set by the deploy script?

```javascript
const fileServerUrlByEnv: {
  dev: 'https://dev.usaspeending.gov/etc'
}

    FILES_SERVER_BASE_URL: fileServerUrlByEng[process.env.FILES_SERVER_BASE_URL]
```